cli/cmd/inject.go

```diff
@@ -156,6 +162,7 @@ func (rt resourceTransformerInject) transform(bytes []byte) ([]byte, []inject.Re
 		conf.AppendPodAnnotations(rt.overrideAnnotations)
 	}
 
+	// Need help from this part
```

Comment:
I need help starting from this part. The annotations are being injected to `conf.workload.Meta.Annotations`. However, the generated JSON is still the same as the input.